tests/josegonzalez/Dotenv/LoaderTest.php

```diff
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Comment:
I mean in terms of other places where we accept "invalid" `.env` syntax, should we also throw exceptions instead?